cmd/buildah/images.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	imagesFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "quiet",
+			Usage: "omit column headings",
+		},
+		cli.BoolFlag{
+			Name:  "qq",
```

Comment:
Depending on the online context, qq means 'crying eyes' or 'just quit the game as you've no skills'.  Perhaps --onlyIDs  or --short or --min?